synthtool/gcp/discogapic_generator.py

```diff
@@ -52,7 +52,7 @@ def java_library(self, service: str, version: str, **kwargs) -> Path:
         return self._generate_code(service, version, "java", **kwargs)
 
     def _generate_code(
-        self, service, version, language, config_path=None, artman_output_name=None
+        self, service, version, language, config_path=None, artman_output_name=None, include_protos=False,
```

Comment:
For python synth.py (the drive to implement this) we use gapic_generator, but I added identical support to discogapic.